### PR TITLE
feat: support GlitchTip integration

### LINKED ISSUE #229: Set up glitchtip for monitoring errors in a deployment
As a security tracker developer, I want to know which errors the system throws in order to be able to address them more effectively. 

- deploy https://glitchtip.com
- register a catch handler

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Via Sentry SDK, we configure a GLITCHTIP_DSN as provided by GlitchTip.

One per environment.

Depends on #261.
Fixes #229.

### CODE REVIEW

**Comment by fricklerhandwerk** [comment]
Let's leave it as a draft before there's also all the deployment code and docs in place.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -283,3 +283,11 @@ If you have your SSH keys set up on the staging environment (and can connect thr
 ### Adding SSH keys
 
 Add your SSH keys to `./staging/configuration.nix` and let existing owners deploy them.
+
+## Operators guidance
+
+### Using a Sentry-like collector
+
+You can set `GLITCHTIP_DSN` as a credential secret with a DSN and this will connect to a Sentry-like endpoint via your DSN.
```
Comment (fricklerhandwerk) [inline CONTRIBUTING.md:291]: What's a DSN, can you please link to a definition? What's Sentry? These names are so generic that we need concrete definitions.

**Review by fricklerhandwerk — APPROVED** [review]
Looks good; just needs a bit more pointers in documentation so we know where to hook into upstream docs for figuring things out.

File: CONTRIBUTING.md (hunk 1 above)
Comment (RaitoBezarius) [reply CONTRIBUTING.md:291]: OK, I will put links.

**Pull request merged**